src/Form/SettingsForm.php

```diff
@@ -313,6 +329,8 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $config->set('schedule_collapse_group', $form_state->getValue('schedule_collapse_group'))->save();
     $config->set('category_collapse_group', $form_state->getValue('category_collapse_group'))->save();
     $config->set('locations_collapse_group', $form_state->getValue('locations_collapse_group'))->save();
+    $config->set('disable_program_search_log', $form_state->getValue('disable_program_search_log'))->save();
```

Comment:
I added default values to config. But we don't need any upgrade path because after upgrade everything should work as before. Any changes will appear only if user changes the settings.

Reply:
Ok, thanks